CQUI.modinfo

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.0">
+<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.01">
```

Comment:
Not sure how the workshop will deal with "01". I would just use a number without zero prefix